### ISSUE TITLE
va-tooltip doesn't work with text node

### ISSUE THREAD

**Issue by rejifald** [nvms/vue-atlas#71]
Hi, va-tooltip doesn't work in case
`<va-tooltip content="Example"> Plain text </va-tooltip>`
I believe it should. WDYT ?

**Comment by nvms** [comment]
looks like you discovered a bug. I'm not sure why this isn't working, because the `Plain text` in the example above gets wrapped in a `span` node element, which should end up being what triggers the tooltip. I'll take a look at it soon

**Comment by rejifald** [comment]
I investigated problem.
And i believe that problem is here https://github.com/nvms/vue-atlas/blob/67d0c8f508115e323a5f2c95b3ea4f4f0531f4cf/src/Mixin/popoverMixin.js#L66
And therefore, i`ve tried to add two or more elements in popover, and it's working only on first element.
I see the solution is:
remove `EventListener.listen(triger, 'mouseenter', ()=>{}) and EventListener.listen(triger, 'mouseleave', ()=>{})` and use vue native `@mouseover="handler()"` on trigger element. It will save u from possible issues related to subscribing and unsubscribing from events.
Finally, something like that
```
    <span ref="trigger" @mouseenter="mouseenter()" @mouseleave="mouseleave()">
        <slot/>
    </span>
```

**Comment by nvms** [comment]
thanks for looking into it. I'll check it out tomorrow and see if I can improve it

**Comment by nvms** [comment]
in the meantime, please feel free to submit a pull request if you are compelled to fix it sooner than that

**Comment by rejifald** [comment]
I`ve provided PR. Please, take a look into this.

**Comment by nvms** [comment]
merged #80 - thanks
